lib/active_force/sobject.rb

```diff
@@ -62,6 +63,9 @@ def self.describe
     def self.build mash, association_mapping={}
       return unless mash
       sobject = new
+
+      not_selected_attributes = sobject.class.fields.reject{|key| mash.keys.include?(key)}
```

Comment:
Do you think `attributes_not_selected` is more readable than `not_selected_attributes`?

Reply:
I like it